src/active_handout_plugins/pdf.py

```diff
@@ -0,0 +1,18 @@
+from markdown.treeprocessors import Treeprocessor
+import xml.etree.ElementTree as etree
+import re
+
+from .utils import AdmonitionVisitor
+
+class PdfAdmonition(AdmonitionVisitor):
+    def visit(self, el):
+        if not 'pdf' in el.attrib['class']:
+            return 
+        
+        img_el = el.find("p/img")
+        if not img_el is None:
```

Comment:
I prefer `if img_el is not None`, but that's fine. Just wanted to express my feelings: ![](https://media.giphy.com/media/RJAjTowsU0K1a/giphy.gif)